README.md

```diff
@@ -19,7 +19,7 @@ Rewrite of the Hypixel Skybot.
 ## Hosting
 
 ### Config.json
-config.json must be filled out correctly in order for the bot to function properly.
+Config.json must be filled out correctly in order for the bot to function properly.
```

Comment:
This is a filename so should remain lowercase

Reply:
yes i made correction